charts/kusion/README.md.gotmpl

```diff
@@ -79,8 +85,8 @@ helm install kusion-release kusionstack/kusion \
 --set server.replicas=3 \
 --set mysql.enabled=true \
 --set mysql.database=kusion \
---set kubeconfig.kubeConfigs.kubeconfig0="$KUBECONFIG_CONTENT1" \
---set kubeconfig.kubeConfigs.kubeconfig1="$KUBECONFIG_CONTENT2"
+--set kubeconfig.kubeConfigs.kubeconfig-0="$KUBECONFIG_CONTENT1" \
```

Comment:
The kubeconfig key naming convention change is consistent here with the previous changes, but again, this should be clearly communicated as a breaking change.